source/base/mu_parser_internal.cc

```diff
@@ -0,0 +1,190 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2005 - 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#include <deal.II/base/mu_parser_internal.h>
```

Comment:
You'll also have to `#include` the muparser header files here.

Reply:
I assume you mean the muParser.h header. If so I don't see the reason. Everything compiles and the tests run successfully on my machine.

Reply:
If you are fine with that I will squash my commits after the last points are resolved.